pkgs/data/themes/flat-remix-gtk/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "flat-remix-gtk";
+  version = "20200718";
+
+  src = fetchFromGitHub {
+    owner = "daniruiz";
+    repo = pname;
+    rev = version;
+    sha256 = "04hv8b512ldjrn4gigna24gdf3finm7j98995npm7lyhnfc4smgf";
+  };
+
```

Comment:
[The gtk2 theme uses the murrine engine](https://github.com/daniruiz/flat-remix-gtk/blob/master/Flat-Remix-GTK-Blue/gtk-2.0/main.rc#L2220). Add the dependence:
```
  propagatedUserEnvPkgs = [ gtk-engine-murrine ];
```